### ISSUE TITLE
Is there any way to run a subquery

### ISSUE THREAD

**Issue by CutePrince30** [NLPchina/elasticsearch-sql#296]
Helo, I want to get the count of username, but it doesn`t work, does it support subquery?

SQL
-----
SELECT count(A.username)
FROM
  ( SELECT username
   FROM tableA
   WHERE operateTime BETWEEN '2016-03-01 00:00:00' AND '2016-07-01 23:59:59'
   GROUP BY username) AS A

ES-SQL-Query
-----
SELECT count(A.*) from (
SELECT 
    username.raw
FROM
    index/tableA
WHERE
    operateTime BETWEEN '2016-03-01T00:00:00Z' AND '2016-07-01T23:59:59Z'
GROUP BY username.raw ) as A

Error
------
<img width="1039" alt="screen shot 2016-09-29 at 4 40 23 pm" src="https://cloud.githubusercontent.com/assets/11158476/18947086/3829ccea-8664-11e6-8ea2-7804ea19c6cc.png">

**Comment by eliranmoyal** [comment]
No, grouping use es aggregations, sub queries is only used for filters
